taccsite_cms/static/site_cms/js/modules/setTargetForExternalLinks.js

```diff
@@ -5,10 +5,12 @@
 const SHOULD_DEBUG = window.DEBUG;
 
 /**
- * Set external links (automatically discovered) to open in new tab
+ * Make links with absolute URLs open in new tab, and:
+ * - add accessible markup
+ * - fix absolute URLs that should be relative paths
  */
 export default function findLinksAndSetTargets() {
-  const links = document.getElementsByTagName('a');
+  const links = document.querySelectorAll('body > :is(header, main, footer) a');
```

Comment:
**Why?** So that CMS Admin UI toolbar links are not affected. _I noticed the placeholder snippet version of this PR was breaking the CMS Admin UI toolbar._